### ISSUE TITLE
New GUI doesn't do anything (no Connect, no Settings, no config view/edit...)

### ISSUE THREAD

**Issue by FlorentBouchy** [jkotra/eOVPN#24]
Since eOVPN GUI update a few days ago, it doesn't work at all anymore: no connection or settings or update. 
The look and feel drastically changed.
The About menu and Keyboard Shortcuts still does work fine, but nothing happens when I click Connect or Settings or Update or the config file.
I tried updating my whole apt/apt-get, uninstalling eOVPN then re-installing it via flatpak following github's instructions exactly (was already installed by flatpak), but no change. 
I'm on Ubuntu 18.04.5 LTS.

If I run eOVPN from a terminal, I can see warnings and errors; for instance when I run from terminal the window opens, no error, but if I click on **Settings** then I get the following error message:
```
$ flatpak run com.github.jkotra.eovpn
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/eovpn/main_window.py", line 212, in <lambda>
    action.connect("activate", lambda x, d: SettingsWindow().show())
  File "/app/lib/python3.8/site-packages/eovpn/settings_window.py", line 278, in show
    self.setup()
  File "/app/lib/python3.8/site-packages/eovpn/settings_window.py", line 171, in setup
    if (password := Secret.password_lookup_sync(self.EOVPN_SECRET_SCHEMA, {"username": username}, None)) is not None:
gi.repository.GLib.Error: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)
```

If I click on **Connect** button then I get the following error message:
```
$ flatpak run com.github.jkotra.eovpn
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/eovpn/main_window.py", line 321, in connect
    manager.connect(self.EOVPN_CONFIG_DIR + "/CONFIGS/" + config)
  File "/app/lib/python3.8/site-packages/eovpn/connection_manager.py", line 40, in connect
    nm_password = Secret.password_lookup_sync(self.EOVPN_SECRET_SCHEMA, {"username": self.get_setting(self.SETTING.AUTH_USER)}, None)
gi.repository.GLib.Error: g-dbus-error-quark: org.freedesktop.DBus.Error.ServiceUnknown (2)
```

EDIT: if I run it with Python debug, I get the following (same output whether --debug 0 or --debug 50)
```
$ flatpak run com.github.jkotra.eovpn --debug 50
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/eovpn/application.py", line 78, in do_command_line
    logging.basicConfig(level=debug_lvl, format='%(levelname)s:%(name)s.py:%(funcName)s:%(message)s')
  File "/usr/lib/python3.8/logging/__init__.py", line 2002, in basicConfig
    root.setLevel(level)
  File "/usr/lib/python3.8/logging/__init__.py", line 1417, in setLevel
    self.level = _checkLevel(level)
  File "/usr/lib/python3.8/logging/__init__.py", line 194, in _checkLevel
    raise ValueError("Unknown level: %r" % level)
ValueError: Unknown level: '50'
```

**Comment by jkotra** [comment]
1st error is caused due to possibly missing package - `libsecret` which is used to securely store password in keying.
`apt install libsecret-1-0`

^ should fix the issue, else post logs here and i will try to spin up a VM and check it out myself.

2nd error is caused due to passing integer level value instead of string (`CRITICAL`, `ERROR` etc..). Ideally it should be able to take any of them, i will fix this in next release. thanks for the bug report.

**Comment by FlorentBouchy** [comment]
Thanks for the quick answer. It doesn't look like it was because of missing libsecret:
`libsecret-1-0 is already the newest version (0.18.6-1).`
What changed in last week's update? I've been using eOVPN everyday for months and it always worked fine; now the GUI changes a lot and it doesn't work anymore.

Also if I run DEBUG with CRITICAL instead of 50, or ERROR etc., then the GUI opens but same issues, and exact same error messages as above.

Note that when I tried to follow your flatpak-builder I got the following errors:
```
$ flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean
Can't load 'dist/flatpak/com.github.jkotra.eovpn.yml': Failed to open file “/home/flo/dist/flatpak/com.github.jkotra.eovpn.yml”: No such file or directory
```

**Comment by FlorentBouchy** [comment]
After a quick look at commit history, I see several changes last weekend, including GTK redesign, NetworkManager by default, etc. I think the NetworkManager by default might be part of the issue why I cannot connect (until last week I had major bugs when I switched between NetworkManager mode and the other one that I already forgot; I had to reinstall eOVPN if I changed this mode by mistake).
Maybe a mix of GTK and this connection mode are what's causing eOVPN not to work anymore on my machine, even after reinstall.

**Comment by jkotra** [comment]
it's definitely related `libsecret`. app is somehow unable to access `org.freedesktop.secrets` bus, the problem is to find out what exactly is causing this.

**Comment by jkotra** [comment]
@FlorentBouchy 

if you wish to downgrade to previous version (`0.41`) for the time being:
```
git https://github.com/jkotra/eOVPN/
git checkout 88b59488d009ec98756779263ceb6202c5d42ac3
flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean
```

**Comment by FlorentBouchy** [comment]
Thanks I'd like to do it but each of these commands generates an error:
```
$ sudo apt-get install git
[sudo] password for flo: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
git is already the newest version (1:2.17.1-1ubuntu0.8).
0 upgraded, 0 newly installed, 0 to remove and 11 not upgraded.

$ git https://github.com/jkotra/eOVPN/
git: 'https://github.com/jkotra/eOVPN/' is not a git command. See 'git --help'.
flo@floBuntu18:~$ git checkout 88b59488d009ec98756779263ceb6202c5d42ac3
fatal: not a git repository (or any of the parent directories): .git

$ flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean
Can't load 'dist/flatpak/com.github.jkotra.eovpn.yml': Failed to open file “/home/flo/dist/flatpak/com.github.jkotra.eovpn.yml”: No such file or directory
```

**Comment by jkotra** [comment]
@FlorentBouchy 

Made a typo, edited my comment now. Try again.

**Comment by FlorentBouchy** [comment]
Ok except for last command:
```
$ flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean
Can't parse 'dist/flatpak/com.github.jkotra.eovpn.yml': <data>:1:6: Parse error: unexpected identifier `app-id', expected value
```

**Comment by jkotra** [comment]
https://github.com/Stremio/stremio-bugs/issues/167#issuecomment-443229865

Uninstall `flatpak` and `flatpak-builder` and follow this: https://flatpak.org/setup/Ubuntu/

Once you have latest version, run the last command again.

**Comment by jkotra** [comment]
```
sudo add-apt-repository ppa:alexlarsson/flatpak
sudo apt update
sudo apt install flatpak flatpak-builder
```

**Comment by FlorentBouchy** [comment]
Ok so I upgraded flatpak-builder.
Now I get this:
```
$ flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean
error: org.gnome.Sdk/x86_64/40 not installed
Failed to init: Unable to find sdk org.gnome.Sdk version 40
```
...and then if I try to solve this error message I get this:
```
$ flatpak install --user org.gnome.Sdk//master
Looking for matches…
error: No remote refs found similar to ‘org.gnome.Sdk//master’
```

**Comment by jkotra** [comment]
`flatpak install --user org.gnome.Sdk//40`

**Comment by jkotra** [comment]
You can also run
`flatpak install --user org.gnome.Sdk`

And choose `40` from available options.

**Comment by FlorentBouchy** [comment]
Tried these already, always the same error message whether I end the command with `.Sdk//master` or `.Sdk//40` or just `.Sdk`.

**Comment by jkotra** [comment]
fixed in `1.05`. Available on flathub, please update :+1:

**Comment by jkotra** [comment]
I've tested this build on Ubuntu 18.04 LTS. works fine. please report if you have any issues.

**Comment by FlorentBouchy** [comment]
Ok thanks, I updated to 1.05 successfully via:
```
$ flatpak update com.github.jkotra.eovpn
Looking for updates…


        ID                             Branch Op Remote  Download
 1. [✓] com.github.jkotra.eovpn.Locale stable u  flathub  7.0 kB / 3.0 MB
 2. [✓] com.github.jkotra.eovpn        stable u  flathub 13.3 kB / 6.4 MB

Updates complete.
```
Then I ran eOVPN via my taskbar icon/shortcut as usual, and the Settings button does work properly now.
**However the Connect button still does nothing.**
If I run eOVPN from the terminal to see the issue when I click Connect button:
```
$ flatpak run com.github.jkotra.eovpn
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/eovpn/main_window.py", line 321, in connect
    manager.connect(self.EOVPN_CONFIG_DIR + "/CONFIGS/" + config)
  File "/app/lib/python3.8/site-packages/eovpn/connection_manager.py", line 48, in connect
    (nm_ca.encode('utf-8')))
AttributeError: 'NoneType' object has no attribute 'encode'
```

Please note that I still cannot run `flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean` or `flatpak install --user org.gnome.Sdk` because of the above errors in my previous messages.

**Comment by jkotra** [comment]
yes, i made a quick fix(https://github.com/jkotra/eOVPN/commit/bb2d3e246dec7141657f11207ba38dfb8739c973) release `1.06`. please update, it should most definitely work now.

**Comment by FlorentBouchy** [comment]
Ok so now there's a new error when I click Connect...
As soon as I click connect, eOVPN closes/crashes immediately.
Running from the terminal I can see this error message:
```
$ flatpak run com.github.jkotra.eovpn
configuration error: option auth-user-pass expects between 0 and 1 argument (line 90)
```
I haven't changed my config file, though.
Line 90 is exactly `auth-user-pass` and has always worked like this, so far.
Lines 89-91 of my config file are:
```
auth SHA1
auth-user-pass
route 192.168.0.0 255.255.252.0
```
I also tried to add my username after `auth-user-pass`, and also my username then password, but it doesn't work either (Connect button is greyed out when I do so; same if I uncheck Authenticate in the Settings).

**Comment by jkotra** [comment]
1. Press Reset in settings and Close.
2. Open eOVPN with `--debug DEBUG` and try to reproduce error.
3. When it crashes, post logs here.

I'd like to communicate with you 1-on-1, can you join #eovpn on Libera Chat IRC server? I will be there.

**Comment by FlorentBouchy** [comment]
Ok that's strange, now it doesn't crash anymore; it seems to connect properly but the connection is not usable... (i.e. I use Remmina to remote desktop through this VPN, but now it doesn't work since last weekend's update).
So the issue seems like it was partly that it needed to re-validate the config file... but when I click the blue Validate and Load button, nothing seems to happen.
I have to close the Settings window and then click Connect (maybe play with Display Flag and Authenticate options until the Connect button becomes available aka not greyed out).
Clicking Connect: I see a thin yellow progress bar while connecting for a few seconds, then it becomes full&green and the Connect button becomes a red Disconnect button that seems to work fine.
But Remmina (which hasn't changed in the last few days) doesn't connect anymore.
```
$ flatpak run com.github.jkotra.eovpn --debug DEBUG
INFO:eovpn.networkmanager.bindings.py:__init__:NM debug = 1
INFO:eovpn.networkmanager.bindings.py:get_version:NM version=b'1.10.6'
INFO:eovpn.networkmanager.bindings.py:get_version:NM openvpn plugin=True
DEBUG:eovpn.networkmanager.dbus.py:watch:<Gio.DBusConnection object at 0x7f7f282da240 (GDBusConnection at 0x561f5575d250)>
DEBUG:eovpn.eovpn_base.py:get_setting:show-flag False
DEBUG:eovpn.networkmanager.bindings.py:get_connection_status:eovpn_nm.is_vpn_running() = False
DEBUG:eovpn.eovpn_base.py:get_setting:last-connected-cursor -1
DEBUG:eovpn.eovpn_base.py:get_setting:remote None
DEBUG:eovpn.eovpn_base.py:get_setting:req-auth False
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user None
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user None
DEBUG:eovpn.eovpn_base.py:get_setting:ca None
DEBUG:eovpn.eovpn_base.py:get_setting:notifications True
DEBUG:eovpn.eovpn_base.py:get_setting:show-flag False
DEBUG:eovpn.eovpn_base.py:set_setting:remote '/home/flo/Desktop/_sharedVM/Lion-STJ.ovpn.zip'
INFO:eovpn.utils.py:download_remote_to_destination:Lion-STJ.ovpn
DEBUG:eovpn.eovpn_base.py:get_setting:req-auth False
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/eovpn/main_window.py", line 50, in row_changed
    if ovpn_is_auth_required(self.EOVPN_OVPN_CONFIG_DIR + row.get_child().get_label()) and self.get_setting(self.SETTING.REQ_AUTH) is False:
AttributeError: 'NoneType' object has no attribute 'get_child'
INFO:eovpn.utils.py:download_remote_to_destination:Lion-STJ.ovpn
DEBUG:eovpn.eovpn_base.py:get_setting:req-auth False
DEBUG:eovpn.eovpn_base.py:set_setting:req-auth true
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'f'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'fl'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'flo'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'flor'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'flore'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'floren'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.b'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.bo'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.bou'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.bouc'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.bouch'
DEBUG:eovpn.eovpn_base.py:set_setting:auth-user 'florent.bouchy'
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'A'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'Ac'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'Act'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'Actu'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'Actua'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'Actual'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'ActualP...'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'ActualPasswordReplaced'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/eovpn/main_window.py", line 50, in row_changed
    if ovpn_is_auth_required(self.EOVPN_OVPN_CONFIG_DIR + row.get_child().get_label()) and self.get_setting(self.SETTING.REQ_AUTH) is False:
AttributeError: 'NoneType' object has no attribute 'get_child'
INFO:eovpn.utils.py:download_remote_to_destination:Lion-STJ.ovpn
DEBUG:eovpn.eovpn_base.py:get_setting:req-auth True
DEBUG:eovpn.eovpn_base.py:set_setting:show-flag true
DEBUG:eovpn.networkmanager.bindings.py:get_connection_status:eovpn_nm.is_vpn_running() = False
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:get_setting:ca None
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:get_setting:auth-pass ActualPasswordReplaced
proxy

ipv4
	address-data : []
	dns : []
	dns-search : []
	method : 'auto'
	route-data : [{'dest': <'192.168.0.0'>, 'prefix': <uint32 22>}]

vpn
	data : {'auth': 'SHA1', 'ca': '/home/flo/.cert/nm-openvpn/Lion-STJ-ca.pem', 'cert': '/home/flo/.cert/nm-openvpn/Lion-STJ-cert.pem', 'cipher': 'AES-128-CBC', 'connection-type': 'password-tls', 'dev': 'tun', 'key': '/home/flo/.cert/nm-openvpn/Lion-STJ-key.pem', 'password-flags': '1', 'ping': '10', 'ping-restart': '60', 'proto-tcp': 'yes', 'remote': 'ActualServerIPReplaced:1194', 'remote-cert-tls': 'server', 'reneg-seconds': '28800', 'username': 'florent.bouchy'}
	secrets : {'password': 'ActualPasswordReplaced'}
	service-type : 'org.freedesktop.NetworkManager.openvpn'

ipv6
	address-data : []
	dns : []
	dns-search : []
	method : 'auto'
	route-data : []

connection
	id : 'Lion-STJ'
	permissions : []
	type : 'vpn'
	uuid : '7966dea4-c4c0-404a-9b2a-b66da5c2b7d7'

** Message: 12:51:15.506: [NM] Connection Added!
** Message: 12:51:15.570: [NM] Connection Connected!
DEBUG:eovpn.eovpn_base.py:set_setting:nm-active-uuid '7966dea4-c4c0-404a-9b2a-b66da5c2b7d7'
DEBUG:eovpn.networkmanager.dbus.py:sub_callback:VpnStateChanged (uint32 2, uint32 1)
0.0
DEBUG:eovpn.networkmanager.dbus.py:sub_callback:VpnStateChanged (uint32 3, uint32 1)
0.35
DEBUG:eovpn.networkmanager.dbus.py:sub_callback:VpnStateChanged (uint32 4, uint32 1)
0.7
DEBUG:eovpn.networkmanager.dbus.py:sub_callback:VpnStateChanged (uint32 5, uint32 1)
DEBUG:eovpn.networkmanager.dbus.py:sub_callback:VPN connected.
DEBUG:eovpn.eovpn_base.py:set_setting:last-connected 'Lion-STJ.ovpn'
DEBUG:eovpn.eovpn_base.py:get_setting:notifications True
DEBUG:eovpn.eovpn_base.py:set_setting:listbox-v-adjust 0.0
DEBUG:eovpn.eovpn_base.py:set_setting:last-connected-cursor 0
DEBUG:eovpn.eovpn_base.py:get_setting:req-auth True
DEBUG:eovpn.eovpn_base.py:get_setting:remote /home/flo/Desktop/_sharedVM/Lion-STJ.ovpn.zip
DEBUG:eovpn.eovpn_base.py:get_setting:req-auth True
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:get_setting:ca None
DEBUG:eovpn.eovpn_base.py:get_setting:notifications True
DEBUG:eovpn.eovpn_base.py:get_setting:show-flag True
INFO:eovpn.settings_window.py:on_password_lookup:<Gio.Task object at 0x7f7f1b1c31c0 (GTask at 0x561f572da0c0)>
DEBUG:eovpn.eovpn_base.py:get_setting:auth-pass ActualPasswordReplaced
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:get_setting:auth-user florent.bouchy
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'ActualPasswordReplaced'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
DEBUG:eovpn.eovpn_base.py:set_setting:auth-pass 'ActualPasswordReplaced'
WARNING:eovpn.settings_window.py:on_password_stored:Password saved as plain text!
```

I'll try to join you on the chat.

**Comment by jkotra** [comment]
Every time you select a config from list, it calls a function to check if authentication is required. If it is required and `Authentication` is toggled off in settings, the button gets greyed out. it's that simple, no black magic involved.

https://github.com/jkotra/eOVPN/blob/97382304515c9f377e3afb07bb2db79f07e3ce59/eovpn/utils.py#L66-L73

**Comment by jkotra** [comment]
> Please note that I still cannot run `flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean` or `flatpak install --user org.gnome.Sdk` because of the above errors in my previous messages.

try running `flatpak install org.gnome.Sdk`

you probably installed flathub repo system wide and `--user` flag is causing the problem.

screenshot from my Ubuntu 18.04 VM:
![Screenshot from 2021-09-08 23-05-47](https://user-images.githubusercontent.com/29590316/132557698-838481db-f6a9-4017-bbf3-4ad6d651a857.png)

**Comment by FlorentBouchy** [comment]
> Every time you select a config from list, it calls a function to check if authentication is required. If it is required and `Authentication` is toggled off in settings, the button gets greyed out. it's that simple, no black magic involved.
> 
> https://github.com/jkotra/eOVPN/blob/97382304515c9f377e3afb07bb2db79f07e3ce59/eovpn/utils.py#L66-L73

Ok the thing is I need to change the Display Flag option to force the button greyout/enable to refresh, even after activating Authenticate option.


> > Please note that I still cannot run `flatpak-builder --user --install build-dir dist/flatpak/com.github.jkotra.eovpn.yml --force-clean` or `flatpak install --user org.gnome.Sdk` because of the above errors in my previous messages.
> 
> try running `flatpak install org.gnome.Sdk`
> 
> you probably installed flathub repo system wide and `--user` flag is causing the problem.
> 
> screenshot from my Ubuntu 18.04 VM:
> ![Screenshot from 2021-09-08 23-05-47](https://user-images.githubusercontent.com/29590316/132557698-838481db-f6a9-4017-bbf3-4ad6d651a857.png)
Ok so removing the `--user` did work properly. I was able to revert to version 0.41 but... it doesn't work anymore(!). Settings button does nothing, I cannot load/import a config file.

So eventually eOVPNversion 1.06 seems like it's working, but... I still cannot make Remmina or ping go through.

Also I cannot join your chat (it seems I would need a pre-existing login/username?)

**Comment by jkotra** [comment]
I'll look into the greyout issue.

--
(0.41)
enable debug mode and look for error when clicking on settings. Post here if you couldn't figure it out. 

I also recommend a clean install: 
`flatpak remove eovpn --delete-data`

And then delete all versions of eovpn installed. Then install 0.41 with flatpak builder. 

--

(1.06)

I have no idea why you can't access Remmina. If the connection is sucessful and you are able to browse the Internet, then job of the app is done. 

Even though EOL of your distro is another 2 years away, upgrading to more recent version would solve many of the problems.

**Comment by FlorentBouchy** [comment]
OK so finally it's working :-)
From eOVPN 1.06 the only remaining issue I had was that my other VPN had a new KillSwitch setting that came activated by default during an update last weekend too (weird coincidence)... so that was causing my remaining issue with Remmina and ping, but eOVPN does work.

For the record I didn't manage to fix eOVPN 0.41 after a full `flatpak remove eovpn --delete-data`... I could open the Setting once, but then I couldn't open them again (I had similar issues when I first installed it a few months ago and I ended up making it work but I don't remember exactly how, was really tricky/glitchy GUI behavior). But I won't get back to this version since 1.06 is fine now, and for remaining glitches I will probably upgrade to Ubuntu 20.04 LTS in the next few months (maybe I'll try Zorin OS 16 instead).
